### ISSUE TITLE
Received "TooManyRequestsException" from StepFunction in AWS Console

### ISSUE THREAD

**Issue by yzhanggithub** [alexcasalboni/aws-lambda-power-tuning#133]
I updated the sample-execution-input.json with arn and payload, ran execute.sh, the output looked like the following. When I looked closer, I found the "TooManyRequestsException" in LambdaFunctionFailed step from StepFunction in AWS Console. How can I tell which request received the throttling error?

Execution started....SUCCEEDED
Execution output: 
{}

Here is the error:
{
  "error": "TooManyRequestsException",
  "cause": {
    "errorType": "TooManyRequestsException",
    "errorMessage": "Rate Exceeded.",
    "trace": [
      "TooManyRequestsException: Rate Exceeded.",
      "    at Object.extractError (/var/runtime/node_modules/aws-sdk/lib/protocol/json.js:52:27)",
      "    at Request.extractError (/var/runtime/node_modules/aws-sdk/lib/protocol/rest_json.js:55:8)",
      "    at Request.callListeners (/var/runtime/node_modules/aws-sdk/lib/sequential_executor.js:106:20)",
      "    at Request.emit (/var/runtime/node_modules/aws-sdk/lib/sequential_executor.js:78:10)",
      "    at Request.emit (/var/runtime/node_modules/aws-sdk/lib/request.js:688:14)",
      "    at Request.transition (/var/runtime/node_modules/aws-sdk/lib/request.js:22:10)",
      "    at AcceptorStateMachine.runTo (/var/runtime/node_modules/aws-sdk/lib/state_machine.js:14:12)",
      "    at /var/runtime/node_modules/aws-sdk/lib/state_machine.js:26:10",
      "    at Request.<anonymous> (/var/runtime/node_modules/aws-sdk/lib/request.js:38:9)",
      "    at Request.<anonymous> (/var/runtime/node_modules/aws-sdk/lib/request.js:690:12)"
    ]
  }
}

**Comment by yzhanggithub** [comment]
Wanted to provide an update -

I was able to get a hello-world lambda working, as well as another lambda we have. I found out that the issue is that the first lambda has reserved concurrency set due to business logic. I found this from checking the lambda, such error didn't seem to show as throttling exception in cloud trail or cloud watch, at least I didn't find useful info there.

Here is the command I used to check, I'll close the issue. The tool works well, thanks for the awesome work!

aws lambda get-function-concurrency \
    --function-name MyLambdaName
